src/h2olog/h2olog.cc

```diff
@@ -226,6 +225,18 @@ static void lost_cb(void *context, uint64_t lost)
     tracer->handle_lost(lost);
 }
 
+template <typename T> static std::string build_cflag_d(const char *name, const T &value)
+{
+    return std::string("-D") + std::string(name) + "=" + std::to_string(value);
```

Comment:
```suggestion
    return build_cflag_d(name, std::to_string(value).c_str());
```

As an alternative, we can omit `.c_str()` if we change the second argtype of the other function definition from `const char *` to `const std::string &`.

Reply:
Refactored these stuff: d27775e6bf48707d47e02700f3051cdcf5ec3c14

Before the above commit, `build_cflag_d(name, T)` and `build_cflag_d(name, std::string)` behaved differently: the former makes `-D$name=$expr` and the latter makes `-D$name="$expr"`. I think this was a kind of overloading abuse. So I've splitted it to `build_cc_macro_expr()` and `build_cc_macro_str()`, and now there's no implementation redundancy.